index.html

```diff
@@ -7,9 +7,22 @@
         <!--[if lte IE 8]>
         <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.6.4/leaflet.ie.css" />
         <![endif]-->
+        <style>
+            .leaflet-left { visibility: hidden; }
+            #time {font-weight: bold; font-size:20px;
+                    color: lightgreen; background-color: #333;
+                    border-radius: 5px;
+                    /* padding-left: 8px; padding-right: 8px;
```

Comment:
This is actually my mistake, but still... The two commented lines need to be deleted.